test/models/opensearch_test.rb

```diff
@@ -67,11 +67,11 @@ class OpensearchTest < ActiveSupport::TestCase
   end
 
   test 'can override index' do
-    # fragile test: assumes opensearch instance with at least one index prefixed with `rdi`
+    # fragile test: assumes opensearch instance with at least one index in the `geo` alias
     VCR.use_cassette('opensearch non-default index') do
-      params = { title: 'data' }
-      results = Opensearch.new.search(0, params, Timdex::OSClient, false, 'rdi*')
-      assert results['hits']['hits'].map { |hit| hit['_index'] }.uniq.map { |index| index.start_with?('rdi') }.any?
+      params = { title: 'bermuda' }
+      results = Opensearch.new.search(0, params, Timdex::OSClient, false, 'geo')
+      assert results['hits']['hits'].map { |hit| hit['_index'] }.uniq.map { |index| index.start_with?('gis') }.any?
```

Comment:
I'm not sure that this ultimately matters, but I note that we are querying a `geo` alias and then checking that index values contain not `geo` but `gis`. Part of me wonders whether the test should compare returned results without and with the `geo` argument, looking for a different composition in the search results - but this likely does enough to confirm that when we ask for the geo alias, we get geo records back.

Reply:
Indexes and aliases for this are different. The alias is indeed `geo`. The indexes start with `gis`. I think this is working as intended even if it isn't entirely clear why.